lua-aws/class.lua

```diff
@@ -29,11 +29,7 @@ local class = (function ()
 			return nil
 		end
 		local __lookup = function(self, method)
-			local m = self.class[method]
-			if not m then	
-				error('no such method: ' .. method .. "\n" .. debug.traceback())
-			end
-			return m
+			return self.class[method]
```

Comment:
I removed the empty check because it failed when you access properties of `nil` value  like `credentials.expireTime`